services/QuillLMS/spec/controllers/api/v1/firebase_tokens_controller_spec.rb

```diff
@@ -21,29 +21,6 @@
       end
     end
 
-    context 'when authenticated via OAuth' do
```

Comment:
It's hard for me to imagine that this entire controller is even in use anymore, but I appreciate cleaning up the tests anyway.